src/applications/disability-benefits/all-claims/pages/incidentLocation.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { PtsdNameTitle } from '../content/ptsdClassification';
+import {
+  incidentLocationSchemas,
+  ptsdLocationDescription,
+} from '../content/incidentLocation';
+
+const { addressUI, addressSchema } = incidentLocationSchemas();
```

Comment:
Should these maybe be in `utils` instead of `content`? I know we're planning on reusing this for 781a so inside the schema file won't really make sense.

Reply:
Yes, good catch